### PR TITLE
Mainkrafte lekse 4

### PR DESCRIPTION
**Pull request by jkunimune15**
novi lekse da yi suje da redite: https://www.reddit.com/r/pandunia/comments/gpb4ud/new_word_for_minecraft_babe_4/

### CODE REVIEW

File: pandunia-lekse.csv [hunk 1]
```diff
@@ -314,6 +316,7 @@ bolivia||||Bolivia||Bolivia|||||||||||||Bolivio|Bolivia|Boliwia
 bombe|||pol:por:spa:tur:bomba, rus:бомба (bomba), deu:fra:bombe, fas:(bomb), eng:bomb, ben:বোমা (boma), hin:बम (bam), urd:(bam), jpn:ボム (bomu), kor:봄 (bom), may:bom|bomb|bombe|bomba|bomba|бомба||炸弹||||बम||bom||bomba|bombo|pommi|bomba
 bon (boni)|||fra:bon, por:bom, spa:bueno, + khm:បុណ្យ (bon), tha:บุญ (bun), tel:పుణ్యము (puṇyamu)|good|bon|bueno|bom|хороший|حَسَن|好|良い|||अच्छा||bagus|mzuri||bona|hyvä|dobry
 bon zare||||good luck||buena suerte||||||||||||||hyvä onni (tuuri, säkä, lykky)|dobry los, szczęście
+bonefete||||blessing (boon)||bendición||благо|||恵み||||||||||
```
Comment (barumau) [inline pandunia-lekse.csv:319]: There is already "barke".

File: pandunia-lekse.csv [hunk 2]
```diff
@@ -601,6 +610,7 @@ duwa|||tur: dua, swa: dua, may: doa, ara: دُعَاء‏ /duʿāʾ/|pray||rezar
 duwe||||prayer||rezo (oración)|||||||||||||preĝo|rukous|modlitwa
 duxe|||fra: douche, deu:Dusche, tur:duş, rus:душ (duš), ara:(duš), spa:por:ducha|shower||ducha|||||||||||||duŝo|suihku|prysznic
 e|||fra:et, por:e, zho:和 (hé)|and|et|y|e|||和|||||||||kaj|ja|i, oraz
+efete|||spa:efecto, por:efeito, fra:effet, eng:effect, deu:Effekt, rus:эффект (effekt), tur:efekt, may:efek|effect (result)|effet|efecto|efeito|эффект||影响|効果||||||||||
```
Comment (barumau) [inline pandunia-lekse.csv:613]: There is already "rezulte".

Comment (jkunimune15) [reply pandunia-lekse.csv:613]: I just missed "rezulte". We can use that.

File: pandunia-lekse.csv (hunk 1 above)
Comment (jkunimune15) [reply pandunia-lekse.csv:319]: I didn't realize **barke** could be used like that. Let's use that.

Actually, if the noun for "blessing" is also used as an interjection for "congratulations!", the noun for "curse" should probably be used as an interjection for "damn!", and I think malorezulte is way too long to be used as a curse. So I propose **lane** (ara:لعنة (laʿna), swa:laana, hau:la'ana, tur:lanet, far:la'net, urd:lānat, may:laknat) as an antonym for **barke**.

Comment (barumau) [reply pandunia-lekse.csv]: Sounds good.

**Pull request merged**